feature/media_details/presentation/src/main/java/com/sanaa/presentation/screen/movieDetails/MovieDetailsScreenInteractionListener.kt

```diff
@@ -22,4 +22,7 @@ interface MovieDetailsScreenInteractionListener {
     fun onDismissSaveToListBottomSheet()
     fun onCreateNewListClick()
     fun onDismissAddListBottomSheet()
+    fun onDismissSnack()
```

Comment:
maybe this one if the snackbar is clickable we should rename it to 
```suggestion
    fun onSnackDismissRequested()
```
or 
```suggestion
    fun onDismissSnackClicked()
```